aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -161,7 +164,7 @@ module AptosFramework::Stake {
     }
 
     /// Rotate the consensus key of the validator, it'll take effect in next epoch.
-    public fun rotate_consensus_key(account: &signer, consensus_pubkey: vector<u8>) acquires ValidatorConfig {
+    public(script) fun rotate_consensus_key(account: &signer, consensus_pubkey: vector<u8>) acquires ValidatorConfig {
```

Comment:
eventually we need to revise all these APIs to validate public keys prior to ending the execution.